lib/puppet/module_tool/skeleton/templates/generator/.rubocop.yml

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
I don't think we have a specified format yet, but voxpupuli has prior-art on this and it appears to be the same https://github.com/voxpupuli/modulesync_config/blob/master/moduleroot/.rubocop.yml 👍